### PR TITLE
remove outdated info from faq

### PR DESCRIPTION
**Pull request by techdocsmith**
Removes outdated information about the "window period". Updates the contact information.

cc: @sthetland , @petermarshallio

This PR has:
- [x ] been self-reviewed.

### CODE REVIEW

**Review by capistrant — APPROVED** [review]
These changes LGTM. The CI failure is not a big deal IMO since the docs job passed. I restarted the other failed 1st phase CI jobs just to be safe

File: docs/ingestion/faq.md [hunk 1]
```diff
@@ -100,20 +91,22 @@ Or, if you use hadoop based ingestion, then you can use "dataSource" input spec
...
+You can also change the query granularity using compaction. See [Query granularity handling](../ingestion/compaction.md#query-granularity-handling).
+
 ## Real-time ingestion seems to be stuck
 
 There are a few ways this can occur. Druid will throttle ingestion to prevent out of memory problems if the intermediate persists are taking too long or if hand-off is taking too long. If your process logs indicate certain columns are taking a very long time to build (for example, if your segment granularity is hourly, but creating a single column takes 30 minutes), you should re-evaluate your configuration or scale up your real-time ingestion.
 
 ## More information
 
-Getting data into Druid can definitely be difficult for first time users. Please don't hesitate to ask questions in our IRC channel or on our [google groups page](https://groups.google.com/forum/#!forum/druid-user).
+Data ingestion for Druid can be difficult for first time users. Please don't hesitate to ask questions the [Druid Forum](https://www.druidforum.org/).
```
Comment (sthetland) [inline docs/ingestion/faq.md:112]:
```suggestion
Data ingestion for Druid can be difficult for first time users. Please don't hesitate to ask questions in the [Druid Forum](https://www.druidforum.org/).
```

**Review by sthetland — COMMENTED** [review]
Just the one nit.

**Review by sthetland — APPROVED** [review]
LGTM

**Pull request merged**